src/GraphicsDrawer.cpp

```diff
@@ -321,6 +321,28 @@ void GraphicsDrawer::_updateScreenCoordsViewport(const FrameBuffer * _pBuffer) c
 	gSP.changed |= CHANGED_VIEWPORT;
 }
 
+void GraphicsDrawer::_updateScreenCoordsViewport2(const FrameBuffer* _pBuffer) const
```

Comment:
`GraphicsDrawer::_updateViewport()` is unused now and should be removed.

Reply:
Done.